listenbrainz/webserver/static/js/src/recommendations/RecommendationCard.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import { get as _get } from "lodash";
+import { faEllipsisV } from "@fortawesome/free-solid-svg-icons";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { getArtistLink, getTrackLink } from "../utils";
+import Card from "../components/Card";
+
+export type RecommendationCardProps = {
+  recommendation: Recommendation;
+  playRecommendation: (recommendation: Recommendation) => void;
+  className?: string;
+  isCurrentUser: Boolean;
+  currentUser?: ListenBrainzUser;
+  newAlert: (
+    alertType: AlertType,
+    title: string,
+    message: string | JSX.Element
+  ) => void;
+};
+
+export default class RecommendationCard extends React.Component<
+  RecommendationCardProps
+> {
+  playRecommendation: (recommendation: Recommendation) => void;
+
+  constructor(props: RecommendationCardProps) {
+    super(props);
+
+    this.playRecommendation = props.playRecommendation.bind(
+      this,
+      props.recommendation
+    );
+  }
+
+  render() {
+    const { recommendation, className } = this.props;
+
+    return (
+      <Card
+        onDoubleClick={this.playRecommendation}
+        className={`recommendation-card row ${className}`}
+      >
+        <div className="col-xs-9">
+          <div className="col-xs-12">
+            <div className="track-details">
+              <p title={recommendation.track_metadata.track_name}>
+                {getTrackLink(recommendation)}
+              </p>
+              <p>
+                <small
+                  className="text-muted"
+                  title={recommendation.track_metadata.artist_name}
+                >
+                  {getArtistLink(recommendation)}
+                </small>
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-xs-3 text-center">
```

Comment:
It looks really wide to me. I think `col-xs-1` would be enough for this small button, no?
And that removes the very wide margin around it.
Ah, and that will require a change for the element before it `col-xs-9` -> `col-xs-11`


<img width="793" alt="Capture d’écran 2020-10-13 à 21 41 33" src="https://user-images.githubusercontent.com/6179856/95908152-eb611700-0d9c-11eb-90b2-d6b747ffdb5f.png">